Add docs about loading heartbeat dashboard

dedemorton opened the pull request:
@urso You might want to add a more interesting screen capture of Heartbeat data. 

BTW, I use "dashboards" plural in case we add more later.

dedemorton commented:
Adds dashboard content for item tracked in https://github.com/elastic/beats/issues/3539 meta issue.

tsg commented:
You can find a somehow more colorful dashboard screenshot here: https://github.com/elastic/beats/pull/3531

Also, the new screenshot for `kibana-created-indexes.png` seems wrong to me. Perhaps you replaced by mistake?

dedemorton commented:
@tsg I've replaced the Heartbeat dashboard. There were a few minor changes to the Kibana UI after you added the dashboard, but I think that having realistic looking data is more important than having the UI match up exactly (it's not too far off). 

The other screen capture, kibana-created-indexes.png, probably looks wrong because I replaced an old screen shot (which showed how to set an index as the default) with a screenshot of the Discover page. The old screen shot was leftover from me copying files to use as a template. We don't tell users to set an index as the default anymore because that happens automatically. So the new screenshot is the right one...it just looks weird you compare it to the old one.
